db/seeds.rb

```diff
@@ -5,8 +5,7 @@
 [
   { name: 'Factual', rank: 1 },
   { name: 'HumanOutsourcer', rank: 2 },
-  { name: 'Geocoder', rank: 3 },
-  { name: 'Burden', rank: 4 }
+  { name: 'Burden', rank: 3 }
```

Comment:
Oops, you're totally right. I took a stab at this (refactoring that Job), but I'd like to review it with you later.